compiler/rustc_error_codes/src/error_codes/E0539.md

```diff
@@ -49,3 +48,29 @@ struct Stable;
 #[rustc_const_stable(feature = "stable_fn", since = "1.39.0")] // ok!
 const fn stable_fn() {}
 ```
```

Comment:
error code upgraded to represent the more generic issue of the structure of an attribute not matching the expected structure